app/src/main/java/de/westnordost/streetcomplete/data/osm/osmquests/OsmElementQuestType.kt

```diff
@@ -75,12 +75,14 @@ interface OsmElementQuestType<T> : QuestType<T> {
      *  any mix-ups that far apart */
     val highlightedElementsRadius: Double get() = 30.0
 
-    /** applies the data from [answer] to the given element. The element is not directly modified,
-     *  instead, a map of [tags] is built */
-    fun applyAnswerTo(answer: T, tags: StringMapChangesBuilder)
+    /** applies the data from [answer] to the element that has last been edited at [timestampEdited].
+     * The element is not directly modified, instead, a map of [tags] is built */
+    fun applyAnswerTo(answer: T, tags: Tags, timestampEdited: Long)
```

Comment:
Would `lastEditTimestamp` be clearer and only a few characters longer? Or `editedTimestamp` or maybe `editTimestamp` is probably slightly better English (in terms of allowing a native speaker to understand what the variable does).

Reply:
I think it is fine. It is the same name as the field with the same name in `Element` class

Reply:
<details><summary> In general, I do not think "better English" is a great reason for renaming variables.</summary>

Code is not English. Programming languages may borrow English words (much like English borrows words from many other languages), but they have a different **grammar**, based more in logic and math. I think that grammar is a much better fit for writing unambiguous, maintainable programs than English.
</details>

So, while `lastEditTimestamp` is better English, I don't think it matters.

However, I still prefer `lastEditTimestamp`, because `timestamp` is almost type information here. You could define a `type Timestamp = Long` and then write `lastEdit: Timestamp` and it would be ~equivalent.